examples/ads.amp.html

```diff
@@ -871,6 +872,12 @@ <h2>Bringhub Mini-Storefront</h2>
       heights="(max-width: 270px) 1280px, (max-width:553px) 640px, 338px">
   </amp-embed>
 
+  <h2>ByPlay</h2>
```

Comment:
I don't see a sample ad here. Is it possible to return an example ad? It can help help us with future regressions, debugging, etc

Reply:
Added flag for sample.

https://github.com/ampproject/amphtml/pull/25663/commits/79420de57ae5cd9dfe39b3f15016b672a31aec04#diff-7f5f58e863202ff16d8d7b9b65b85391R878